client/.storybook/main.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  "stories": [
+    "../src/**/*.stories.mdx",
```

Comment:
Duplicated code from #52 

Please fix the base of the branch before I review. Thanks!